docs/extend/plugins.md

```diff
@@ -34,7 +34,7 @@ Follow the instructions in the plugin's documentation.
 The sections below provide an inexhaustive overview of available plugins.
 
 <style>
-#content tr td:first-child { white-space: nowrap;}
+#DocumentationText  tr td:first-child { white-space: nowrap;}
```

Comment:
oh! wait; you're looking on GitHub :smile: that's expected. This style isn't used on GitHub, but these markdown files are used to generate the online documentation.

If you run `make docs` from the root of the repository, then the documentation is built (in a container). You can then access `http://<your docker host>:8000` to view the rendered documentation.

I just checked out your PR, and ran `make docs`, and it looks good:

![screen shot 2016-07-15 at 13 55 50](https://cloud.githubusercontent.com/assets/1804568/16873492/f937a4dc-4a93-11e6-9c30-027a271ed10a.png)